TTS/bin/train_glow_tts.py

```diff
@@ -538,8 +538,16 @@ def main(args):  # pylint: disable=redefined-outer-name
     num_params = count_parameters(model)
     print("\n > Model has {} parameters".format(num_params), flush=True)
 
-    if 'best_loss' not in locals():
+    if args.restore_step == 0 or not args.best_path:
         best_loss = float('inf')
+        print(" > Starting with inf best loss.")
+    else:
+        print(args.best_path)
+        best_loss = torch.load(args.best_path,
+                               map_location='cpu')['model_loss']
+        print(f" > Starting with loaded last best loss {best_loss}.")
+    keep_best = c.get('keep_best', False)
+    keep_after = c.get('keep_after', 10000)  # void if keep_best False
```

Comment:
What do keep_best and keep_after do?

Reply:
I've commented it in the configs. Basically `keep_best` keeps all the best models if `current_step > keep_after`.